src/common/Section/styled.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
```

Comment:
section tag would fit better

Reply:
I guess I like "Wrapper" so much that I'm overusing it;D, changed to SectionTag:)